source/h1_encoder.c

```diff
@@ -351,110 +356,27 @@ int aws_h1_encoder_start_message(
         return aws_raise_error(AWS_ERROR_INVALID_STATE);
     }
 
-    /* Can start writing head next */
     encoder->logging_id = log_as_stream;
     encoder->message = message;
-    encoder->state = AWS_H1_ENCODER_STATE_HEAD;
-    encoder->progress_bytes = 0;
-
-    return AWS_OP_SUCCESS;
-}
-
-static int s_h1_encoder_process_content_length_body(struct aws_byte_buf *dst, struct aws_h1_encoder *encoder) {
-    while (dst->capacity >= dst->len) {
-        if (dst->capacity == dst->len) {
```

Comment:
Github was doing such a terrible job showing the diffs on this function I just moved it below to a better place.